examples/multimodal/utils/protocol.py

```
@@ -23,12 +23,14 @@
 from typing_extensions import NotRequired
 from vllm.inputs.data import TokensPrompt
 from vllm.multimodal.inputs import MultiModalUUIDDict  # noqa: F401
```

Ruff (0.13.3)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)